Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -972,6 +973,36 @@ void VolumeVerifier::CheckMisc()
                              "The CRC32 of this file might match the CRC32 of a good dump even "
                              "though the files are not identical."));
     }
+
+    if (StringBeginsWith(game_id_unencrypted, "R8P"))
+      CheckSuperPaperMario();
+  }
+}
+
+void VolumeVerifier::CheckSuperPaperMario()
+{
+  // https://bugs.dolphin-emu.org/issues/11900
```

Comment:
I know (or at least hope!) our tracker is here to stay, but would you mind including a line or two describing the problem for the case where someone cannot access the tracker (or is simply too lazy to do so?)

Reply:
I don't think such a comment would add anything that isn't already clear from the user-facing message and maybe the line of code that specifies which file we're checking. (Unless the fact that this happens with disc images created by currently existing versions of NKit is important to note in the source code?)

Reply:
I mainly meant the fact that this game (and perhaps others?) will potentially read more data than specified by the size parameter, which _may_ result in issues if they're not zero. The rest is clear from the code and the message.